### DISCUSSION TITLE
cluster threshold

### DISCUSSION THREAD

**Discussion by msiemionCalistapw** [moj-analytical-services/splink#761] [Q&A]
What is the meaning of the .5 threshold in the code below?

cluster = .5
df_clusters = linker.cluster_pairwise_predictions_at_threshold(df_predict, cluster)

**Comment by RobinL** [comment]
https://moj-analytical-services.github.io/splink/linker.html#splink.linker.Linker.cluster_pairwise_predictions_at_threshold
